// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package autofill;

option optimize_for = LITE_RUNTIME;

message StrikeData {
  // Total number of strikes thus far.
  optional int32 num_strikes = 1;

  // Timestamp of when a strike was last added or removed
  // for this entry. Can be used to expire strikes later.
  optional int64 last_update_timestamp = 2;
}
